library/core/src/clone.rs

```diff
@@ -38,7 +38,25 @@
 
 mod uninit;
 
-/// A common trait for the ability to explicitly duplicate an object.
+/// A common trait that allows explicit creation of a "duplicate" value.
```

Comment:
No need for scare quotes. The value is in fact duplicated.
```suggestion
/// A common trait that allows explicit creation of a duplicate value.
```